### PR TITLE
Add Security Overview

### PR DESCRIPTION
**Pull request by Starefossen**
Add Security Overview for NAIS

- [x] Shared responsibility
- [x] Security Principles
- [x] Team isolation
- [x] Build-phase
    - [x] Secrets
    - [x] External dependencies
- [x] Deploy-phase
    - [x] SLSA
    - [x] Security Policies
- [ ] Operate-phase
    - [x] Access control
    - [x] User authentication
    - [x] Network security
    	- [x] Cloud Armor
    	- [x] NetPols
    	- [x] Linkerd
    - [x] Logging

## For discussion

- Threat models
- References to sikker.nav.no
- References to TryggNok

### CODE REVIEW

**Review by jksolbakken — APPROVED** [review]

File: docs/security/README.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    secure manner. This document describes on a high level how NAIS achieves this and what is expected of those who use NAIS.
+---
+
+# Security in NAIS
+
+NAIS is a platform for building, deploying and operating applications in a
+secure manner. This document describes on a high level how NAIS achieves this
+and what is expected of those who use NAIS.
+
+## Shared responsibility
```
Comment (Reasonable-Solutions) [inline docs/security/README.md:13]:
```suggestion
## Responsibility Segregation
```

Also, consider titlecasing headings

File: docs/security/README.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+multiple layers of security controls, so that if one layer fails, the next layer
+will provide protection.
+
+1. **Least privilege** - NAIS is built on the principle of least privilege. This
+means that every component of the platform is given the minimum amount of
+privileges necessary to perform its function. This is achieved by using
+role-based access control (RBAC) and by using the principle of separation of
+duties.
+
+1. **Secure by default** - NAIS is built on the principle of secure by default.
```
Comment (Reasonable-Solutions) [inline docs/security/README.md:37]: This section says very little by saying very little. 
Needs more content

File: docs/security/README.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+This means that every component of the platform is configured to be secure by
+default. This is achieved by using secure defaults and by using the principle of
+least privilege.
+
+1. **Secure by design** - NAIS is built on the principle of secure by design.
```
Comment (Reasonable-Solutions) [inline docs/security/README.md:42]: 1. **Secure by design** - NAIS is built on the principle of secure by design.
This means that every component of the platform is designed to be secure by
design. This is achieved by using secure design patterns 

both this one and secure by default could be just "the principle of least privileges is used"

File: docs/security/README.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+This means that every component of the platform is designed to be secure by
+design. This is achieved by using secure design patterns and by using the
+principle of least privilege.
+
+## Team isolation
+
+NAIS is built on the principle of team isolation. This means that every team has
```
Comment (Reasonable-Solutions) [inline docs/security/README.md:49]: This structure seems to be repeating a lot. Maybe we could make a table of all of those? 
e.g 
| Principle | Meaning | Achieved by

**Pull request merged**